### DISCUSSION TITLE
How to add third-party libraries-SPM?

### DISCUSSION THREAD

**Discussion by it-senior-developer** [skiptools/skip#261] [Q&A] [answered]
As I understand it, Skip does not support all ibraries-SPM? Do I need to register them with you?

**Comment by marcprux** [comment, marked as answer]
You can use any SwiftPM dependencies that you like, but if they are not Skip-enabled, then you will only be able to use them on the iOS side of your app, and you will need to find an alternative for the Android side. 

This is explained at https://skip.tools/docs/dependencies/
